src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -838,26 +834,6 @@ GetModifyConnections(Task *task, bool markCritical, bool noNewTransactions)
 		multiConnection = GetPlacementListConnection(connectionFlags, placementAccessList,
 													 NULL);
 
-		/*
```

Comment:
Would be a good idea. I think we can honour multi_shard_commit_protocol as well.